python/cudf/cudf/core/series.py

```diff
@@ -510,11 +510,12 @@ def __init__(
             data = {}
         if dtype is not None:
             dtype = cudf.dtype(dtype)
-
+        atts = {}
         if isinstance(data, (pd.Series, pd.Index, Index, Series)):
             if copy and not isinstance(data, (pd.Series, pd.Index)):
                 data = data.copy(deep=True)
             name_from_data = data.name
+            atts = getattr(data, "attrs", {})
```

Comment:
question: Why in the series case are we trying to use the attrs from the input and in the dataframe case we just end up assigning `self._attrs = {}`?